### PR TITLE
Add plex

### PR DESCRIPTION
**Pull request by stavros-k**

### CODE REVIEW

File: ix-dev/charts/plex/app.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: plex
+train: charts
```
[the remark this answers is not in the document]
Comment (stavros-k) [reply ix-dev/charts/plex/app.yaml:2]: I think it was simplifying the validation, but will be better to let @sonicaj explain.

**Review by sonicaj — CHANGES_REQUESTED** [review]
Migration is not executable and you are missing shebang as well which would result in it not executing

**Comment by stavros-k** [comment]
> Migration is not executable and you are missing shebang as well which would result in it not executing

We had validations in place for that on charts. Fixed for now, we should re-add the validation on apps too!

File: ix-dev/stable/plex/templates/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{% set plex_container_name = "plex" %}
+{% set plex_images = {
+  "plex_image": "plexinc/pms-docker:1.40.2.8395-c67dce28e",
```
Comment (Qubad786) [inline ix-dev/stable/plex/templates/docker-compose.yaml:3]: How are image updates going to be handled?

**Review by Qubad786 — APPROVED** [review]

**Review by sonicaj — APPROVED** [review]

**Pull request merged**